tooling-docker/src/test/java/org/jboss/eap/qe/ts/common/docker/DockerTest.java

```diff
@@ -0,0 +1,42 @@
+package org.jboss.eap.qe.ts.common.docker;
+
+import io.restassured.RestAssured;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import static org.hamcrest.Matchers.containsString;
+
+public class DockerTest {
+
+    @ClassRule
+    public static Docker wildflyContainer = new Docker.Builder("wildfly", "jboss/wildfly")
```

Comment:
Do we really want to pull the latest vs. one specific version (which can be cached locally) ?

Reply:
+1 as this is just testing tooling, having specific version which can be cached makes sense. Fixed